ocm/charter.md

```diff
@@ -54,7 +57,12 @@ for actual data transfer.
 
 OCM does not aim to address identity federation (e.g., OIDC or SAML);
 rather, it leverages existing protocols for identity and authentication
-where appropriate.
+where appropriate.  OCM also does not propose to create interoperable 
+protocol flows to establish trust relationships where there are none 
+before; instead the WG will explain how trust relationships set up 
+out-of-band (i.e. configuration of server settings or allow lists) 
+are required for secure sharing.
+
 
 Importantly, OCM is **not** a competitor to user-oriented messaging or
```

Comment:
Do we really need to have this text here?

Reply:
Rewording as you suggested @anewton1998 works for me. I was initially puzzled why specific "social ..." protocols were listed.